staff/jordi-pujol/cv/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Jordi_cv</title>
+</head>
+<body>
+    <header>
+        <h1>Jordi Pujol</h1>
+        <img src="https://img.elcomercio.pe/files/article_content_ec_fotos/uploads/2018/10/06/5bb8ce60353c5.jpeg" alt="">
+        <ul>
+            <li><h2>Web Designer Director</h2></li>
+            <li><h2>E-mail:ddddddd@gmail.com</h2></li>
+            <li><h2>Telephone: 83723984729</h2></li>
+        </ul>
+    </header>
+
+    <main>
+        <br>
+        <section><h2>Profile</h2>
+            <ul>
+                <li>Progressively evolve cross-platform ideas befor impactful infomediaries</li>
+            </ul>
+        </section>
+        <br>
+        <section><h2>Skills</h2>
+            <ol>
+                <li><h4>Web Design</h4></li>
+                    <p>ioasdmjoisadoijsaoidjsaoidjaoid <br> doiaoidjaiodajdoi</p>
+                <li><h4>Interface Design</h4></li>
+                    <p>ioasdmjoisadoijsaoidjsaoidjaoid <br> doiaoidjaiodajdoi</p>
+                <li><h4>Project Direction</h4></li>
+                    <p>ioasdmjoisadoijsaoidjsaoidjaoid <br> doiaoidjaiodajdoi</p>
+            </ol>
+        </section>
+        <br>
+        <section><h2>Technical</h2>
+            <ol>
+                <li>XHTML</li>
+                <li>Jquery</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>Linux</li>
+            </ol>
+        </section>
+        <br>
+        <section><h2>Experience</h2>
+            <ol>
+                    <li><h3>2005-2007  Facebook</h3></li>
+                        <p><h4>Senior Interface Designer</h4><br> Intrisicly enable optimal core competencies through corporate relationships. <br>Conveniently initiate virtual paradigms top-line convergence</p>
```

Comment:
Jordi, you can make nested <ul> or <ol> instead of making <p> and <br>.

Reply:
Thanks! I'll change it right now!